app/queries/app/global.ts

```diff
@@ -21,8 +21,12 @@ export const getDeviceToken = async (appDatabase: Database): Promise<string> =>
     }
 };
 
+export const queryGlobalValue = (appDatabase: Database, key: string) => {
```

Comment:
I do have an upcoming PR that will get rid of the use of `appDatabase` and instead use `const {database} = DatabaseManager.getAppDatabaseAndOperator();`

can you make that change here or do you want me to take care of that so that you don't have to make further changes? I think is best if I take care of it